src/schema/rules/checks/fmap.yaml

```diff
@@ -49,3 +49,16 @@ MagnitudeFileWithTooManyDimensions:
     - nifti_header != null
   checks:
     - nifti_header.dim[0] == 3
+
+EPIWithBValsHasHighBVals:
+  issue:
+    code: EPI_WITH_BVALS_HAS_HIGH_BVALS
+    message: |
+      EPI field maps with b-values should not include high b-values (in essence, b-values > 100).
+    level: error
+  selectors:
+    - datatype == "fmap"
+    - suffix == "epi"
+    - '"bval" in associations'
+  checks:
+    - max(associations.bval.rows[0]) > 100
```

Comment:
@effigies not sure if this works with the syntax.

Reply:
Do we care about requiring `max() < 100` or `min() < 100`? I thought the idea was that we should be able to identify b0 volumes, not that they all must be.

Reply:
@mattcieslak can correct me, but I thought that epi scans used for distortion correction would _all_ effectively be b0 volumes, unlike the DWI scans.

Reply:
@tsalo the issue is that some vendors require that a non b=0 image be collected so there are usually a couple b=0s and one or move b>0s. The reason the bvals are so useful here is so we can identify which volumes to use for sdc

Reply:
In that case @effigies has it right. I'll make the changes to look for at least one low bval instead.